.git-commit-template

```diff
@@ -5,5 +5,38 @@ Explanation
 Resolves:
 https://github.com/SSSD/sssd/issues/XXXX
 
+# If a release note is required, choose one of the tags and place it here.
+# See the description bellow for more tags and information.
+:relnote: Generic release note.
+:feature: New feature desription.
+:bug: Notable bug desription.
+
 # Try to keep the subject line within 52 chars ----|
 # Also please try to not exceed 72 characters of length for the body --|
+#
+# *** Release notes ***
+#
+# Release notes for new versions are automatically generated from the
+# information provided in commit messages. Please keep the formatting of
+# resolved tickets exact (Resolves:/NEWLINE/ticket_url). You can specify
+# multiple tickets, one ticket per line.
+#
+# You can also provide short description of the fix or new feature for
+# the release notes using the following tags:
+#
+# :relnote: Generic release note.
+# :feature: New feature desription.
+# :bug: Notable bug desription.
+# :packaging: Packaging change description.
+#
+# :newoption: option_name, manpage_name
+# :deloption: removed_option_name
+# :newdefault: option_name, old_value, new_value
```

Comment:
How to specify `option_name`? Via "full path"?

For example:
`sssd:certificate_verification:ocsp_dgst`
or
`ocsp_dgst`
?

Reply:
Example: https://github.com/SSSD/sssd/pull/5280/commits/b8d7e5643ca8f2ccc1375ed833916e52f2b961a5

Reply:
Just ocsp_dgst is enough, I think.

Reply:
Hmm, I think we should actually get rid of this and use a plain text as in other options.

The reason I introduced it like this `:newoption: name, manpage` is because I used to put options in the release note like `Added option xyz, see man xyz`, but this was just because I was lazy and did not want to come with a meaningful description as I known little about the new options.

However if it gets written by the author, it makes sense to add more information on what it does and why it was done. E.g. `Default value of client_idle_timeout changed from 60 to 300 seconds for KCM, this allows more time for user interaction (e.g. during kinit)`. Also there are so many use cases that we can't cover them all like this - one can add new config section, default value specific to one responder, different meaning in different sections, ... so the benefit of unifying the message format is lost in the complexity.